praw/reddit.py

```diff
@@ -556,6 +556,7 @@ def _objectify_request(
         method: str = "",
         params: Optional[Union[str, Dict[str, str]]] = None,
         path: str = "",
+        json=None,
```

Comment:
I think we should keep the current ordering.

```suggestion
        json=None,
        method: str = "",
        params: Optional[Union[str, Dict[str, str]]] = None,
        path: str = "",
```